Makefile.PL

```diff
@@ -9,63 +9,66 @@
 require 5;
 
 use strict;
-use ExtUtils::MakeMaker;
+use ExtUtils::MakeMaker 6.64;
 
 WriteMakefile(
-  NAME		=> 'Pod::Simple',
-  VERSION_FROM	=> 'lib/Pod/Simple.pm',
-  ABSTRACT_FROM	=> 'lib/Pod/Simple.pod',
-  PREREQ_PM	  	=> {
-   'Text::Wrap' => '98.112902',
-   'Pod::Escapes' => '1.04',
+    NAME          => 'Pod::Simple',
+    VERSION_FROM  => 'lib/Pod/Simple.pm',
+    ABSTRACT_FROM => 'lib/Pod/Simple.pod',
+    TEST_REQUIRES => {
+        'Sub::Util' => 1.55,
```

Comment:
@karenetheridge notice the extra requirement for `Sub-Util`
are you saying we need it also as a regular `PREREQ_PM` even if it's only use by unit tests?

Reply:
ok great; I missed the test_requires update.